rpcs3/evdev_joystick_handler.cpp

```diff
@@ -13,560 +17,756 @@
 #include <cstdio>
 #include <cmath>
 
-evdev_joystick_config g_evdev_joystick_config;
-
-evdev_joystick_handler::evdev_joystick_handler() {}
+evdev_joystick_handler::evdev_joystick_handler()
+{
+	// Define border values
+	THUMB_MIN = 0;
+	THUMB_MAX = 255;
+	TRIGGER_MIN = 0;
+	TRIGGER_MAX = 255;
+	VIBRATION_MIN = 0;
+	VIBRATION_MAX = 65535;
```

Comment:
I used the values defined by documentations and tutorials and I think it is more understandable seeing real numbers. (I don't really know if the vibration max is correct since I didn't implement evdev haptics)

Reply:
ok